config.py

```diff
@@ -200,7 +201,7 @@ class Config:
             "offline_warning_enabled": False,
             "publish_benchmark_results": True,
             "max_builds": 1,
-            "build_timeout": 90,
+            "build_timeout": 270,
```

Comment:
Since that test run took 142m, this can probably be lowered
```suggestion
            "build_timeout": 180,
```